src/stylesheets/components/_navigation.scss

```diff
@@ -419,7 +419,7 @@
     flex-direction: column;
 
     @include media($nav-width) {
-      display: initial;
+      display: block;
```

Comment:
I wonder if it'd be useful to write:

```css
  /* For IE11 */
  display: block;
  /* For newer browsers */
  display: initial;
```

That way the `display: initial` would still be set for all browsers that actually support that rule.

However, I'm unfamiliar with the CSS in question, so I don't actually know if `display: block` is all that's needed for things to look just like they do with `display: initial`... If it is, then I agree that *only* using `display: block` is the way to go!